libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.html

```diff
@@ -63,6 +64,11 @@
   >
   </iframe>
 </div>
+<div skyId #hintTextEl="skyId">
```

Comment:
Same advice as this: https://github.com/blackbaud/skyux/pull/2252/files#r1581371417

Reply:
updated